src/troute-config/troute/config/output_parameters.py

```diff
@@ -4,13 +4,16 @@
 from typing_extensions import Literal
 from .types import FilePath, DirectoryPath
 
-streamOutput_allowedTypes = Literal['.csv', '.nc', '.pkl']
+streamOutput_allowedTypes = Literal['.csv', '.nc', '.pkl', '.parquet']
```

Comment:
`streamOutput_allowedTypes` is specific for the `stream_output` class of parameters. It looks like you've created a new class, `ParquetOutput` that doesn't use this. Is this change needed?

Reply:
Thank you for pointing this out. I have removed `.parquet` from `streamOutput_allowedTypes`